rules/cross-platform/privilege_escalation_sudo_buffer_overflow.toml

```diff
@@ -4,7 +4,7 @@ integration = ["endpoint"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/06/22"
+updated_date = "2024/01/05"
```

Comment:
Are we updating all dates to today (17th)?

Reply:
Only when the rule had an update should the date reflect.